src/main/java/dev/openfeature/sdk/Structure.java

```diff
@@ -134,7 +138,9 @@ default <T extends Structure> Map<String, Value> merge(Function<Map<String, Valu
      */
     static Structure mapToStructure(Map<String, Object> map) {
         return new MutableStructure(map.entrySet().stream()
-            .filter(e -> e.getValue() != null)
```

Comment:
We were FILTERING null values before. This resulted in maps with null entries not having converted (empty) `Values`. I think that's wrong. With this change, all nulls in maps are converted to null-containing `Values` and vice-versa.